l10n/templates/strings.properties

```diff
@@ -127,15 +132,23 @@ settings_pane_topstories_options_sponsored=Show Sponsored Stories
 # LOCALIZATION NOTE (edit_topsites_*): This is shown in the Edit Top Sites modal
 # dialog.
 edit_topsites_button_text=Edit
+# remove this string after 60:
```

Comment:
This is hardly relevant for localizers. Can we put this information somewhere else?

Reply:
Is there some way to keep it in-line but not show up for localizers? Otherwise, we could just put the ids in some bug and hope the list doesn't get out of date if some string happens to get reused and should be removed from the list of to-be-removed ?

Reply:
No, comments will show up in Pontoon. Either we make them understandable, e.g. `This string will be removed in Firefox 61`(which might have the side effect of people ignoring them), or add the list to a bug. I'd be for the latter, since it's an implementation detail.

Reply:
Removed comments and listed as part of the commit message that gets copied to https://bugzilla.mozilla.org/show_bug.cgi?id=1433209